fw/tls.c

```diff
@@ -797,6 +797,7 @@ tfw_tls_conn_send(TfwConn *c, TfwMsg *msg)
 {
 	int r;
 	TlsCtx *tls = tfw_tls_context(c);
+	int ss_flags = msg->ss_flags;
```

Comment:
Don't we need `READ_ONCE()` here to make sure that the compiler won't optimize out the temporal variable?

Reply:
We no't need READ_ONCE, because as i can see the problem is that message destroyed in ss_do_send callback which is called after ss_send on other cpu